packit_service/worker/handlers/github_handlers.py

```diff
@@ -505,25 +522,39 @@ class GitHubPullRequestCommentCoprBuildHandler(CommentActionHandler):
     event: PullRequestCommentGithubEvent
 
     def run(self) -> HandlerResults:
-        user_can_merge_pr = self.event.project.can_merge_pr(self.event.user_login)
-        if not (user_can_merge_pr or self.event.user_login in self.config.admins):
-            self.event.project.pr_comment(
-                self.event.pr_id, PERMISSIONS_ERROR_WRITE_OR_ADMIN
-            )
-            return HandlerResults(
-                success=True, details={"msg": PERMISSIONS_ERROR_WRITE_OR_ADMIN}
+        if isinstance(
+            self.event, (PullRequestCommentGithubEvent, MergeRequestCommentGitlabEvent)
+        ):
```

Comment:
What is the `else` variant here? Can we add some comment about that? (It will go silently without permissions check so we might want to be more explicit here.)

Reply:
Yeah! I have removed that check as we are already checking the events in pre-check.